app/helpers/physical_server_helper/textual_summary.rb

```diff
@@ -1,10 +1,16 @@
 module PhysicalServerHelper::TextualSummary
   def textual_group_properties
-    %i(name model product_name manufacturer machine_type serial_number ems_ref)
+    TextualGroup.new(
+      _("Properties"),
+      %i(name model product_name manufacturer machine_type serial_number ems_ref)
+    )
   end
 
   def textual_group_relationships
-    %i(host)
+    TextualGroup.new(
+      _("Relationships"),
+      %i(host)
```

Comment:
The host relationship is expressed in https://github.com/ManageIQ/manageiq/blob/master/app/models/physical_server.rb#L13